Gopkg.toml

```diff
@@ -0,0 +1,56 @@
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/agl/ed25519"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/btcsuite/btclog"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/btcsuite/go-socks"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/btcsuite/goleveldb"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/btcsuite/websocket"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/btcsuite/winsvc"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/davecgh/go-spew"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/dchest/blake256"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/decred/dcrrpcclient"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/decred/dcrutil"
+
+[[constraint]]
+  name = "github.com/jessevdk/go-flags"
+  version = "1.3.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/jrick/bitset"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/jrick/logrotate"
+
```

Comment:
I don't see a reference to `golang.org/x/sys` although it's in the lock file.

Reply:
it's a transitive dep, not a constraint.

Reply:
Good call.  Indeed it is.